README.md

```diff
@@ -1,40 +1,56 @@
 # Library Room Auto-Scheduler
+
 For Temple students who need to schedule rooms in Charles Library farther in advance, the Library Room Auto-Scheduler is a room booking app that allows you to automatically schedule rooms as soon as they’re available. Unlike Charles’s own scheduler, our product allows proactive students to book rooms more than 2-3 days in advance without needing to navigate through the library’s appointment interface.
 
 ![This is a screenshot.](images/login.png)
 ![This is a screenshot.](images/booking.png)
 
 # How to run
+
 - For MacOS users :
-- Download the latest zip from the Release section on the right on GitHub. 
+- Download the latest zip from the Release section on the right on GitHub.
 - Extract the bookRoom.dmg file from the zip and open it
 - Right click on the bookRoom.app file inside and click 'Open'. A popup will be appear claiming the file is not safe.
 - Repeat the previous step. This time, the popup should have the option 'Open'. Click it and the application should load.
 
 # How to contribute
-Follow this project board to know the latest status of the project: [http://...]([https://github.com/cis3296s22/libraryroomscheduler/projects/2])  
 
-### How to build
-- Use this github repository
-- Use Visual Studio Code
-- Install:
+Follow this project board to know the latest status of the project: [http://...]([https://github.com/cis3296s22/libraryroomscheduler/projects/2])
 
-```
-pip3 install selenium
-pip3 install kivy
-pip3 install webdriver_manager
-pip3 install pyinstaller
-```
-- Run main.py to login and book a room through the command line
-- Run bookRoom.py to login and book a room with a GUI 
-- In both programs, the user will be prompted for their TU credentials to login and they'll be asked to specify the details of their booking.
-- main.py opens the browser to show the automation of the login/booking process, but it won't make the reservation. 
+## How to build
+
+_Note: Visual Studio Code is the recommended editor for the Library Room Auto-Scheduler_
+
+- Python 3.8 or higher is recommended to contribute
+- Clone this repo
+- (Recommended) Create a virtual environment
+  - In the main folder of this project, execute `python3 -m venv venv` on the command line to create the virtual environment
+  - Activate it!
+    - Mac/Linux: `source ./venv/bin/activate`
```

Comment:
Change to `source ./venv/bin/activate.csh`

Reply:
@ccho-0508 are you using the C shell? That's the only reason you would have to use the one with the `.csh` extension. I'll update the README to include both

Reply:
looks like that was the case >:(